_posts/courses/earth-analytics/04-vector-data-gis-r/review-materials/2017-07-25-plot03-interactive-maps-leaflet-R.md

```diff
@@ -99,7 +101,7 @@ Next, import and explore the data.
 # import roads
 plot_locations <- readOGR("data/week-04/california/SJER/vector_data/SJER_plot_centroids.shp")
 ## OGR data source with driver: ESRI Shapefile 
-## Source: "data/week-04/california/SJER/vector_data/SJER_plot_centroids.shp", layer: "SJER_plot_centroids"
+## Source: "/Users/lewa8222/Dropbox/earth-analytics/data/week-04/california/SJER/vector_data/SJER_plot_centroids.shp", layer: "SJER_plot_centroids"
```

Comment:
uh oh... drop box is evil but in this case made life easier. need to build a data download scripts for this course if i ever teach it again to avoid the evil dropbox!!